### ISSUE TITLE
NVDA does not read the Contents Item key (Alt text) for Link Annotations

### ISSUE THREAD

**Issue by FvdGDTP** [nvaccess/nvda#15228]
PDF/UA requires Annotations to have a description, in the old versions of Acrobat you could Go to the Tag Properties and fill in the Alt Text. That would place the Alt text in the Alt Text Code field in the Tag, the same place as with images read by NVDA (and Jaws).

However, PDF/UA requires it not to be ’standard’ Alt Text (like images) but in the Contents Item Key.

InDesign writes it there.

In the latest version of Acrobat Pro the Alt Text files have now split into two entries: one for images and one for Link annotations! The Link annotation Alt Text is now written also solely in the Contents Item key! The ‘image Alte Text’ field is greyed out.

However, NVDA (nor Jaws) does not read (voice) the Contents Item key where it should for Link annotations…!

So in previous versions of Acrobat we violated PDF/UA rules but things worked, now Acrobat breaks that and will not revert back because it is indeed the right correct way to put it in the Contents Item key, this won’t ‘change back’. 
So we need NVDA (and yes, Jaws) to catch up with this reality...

**Comment by Adriani90** [comment]
this is the same like #15229, I am closing this one.
